## <center> Практическое задание к уроку № 2

___

**1. Вероятность того, что стрелок попадет в мишень, выстрелив один раз, равна 0.8. Стрелок выстрелил 100 раз. Найдите вероятность того, что стрелок попадет в цель ровно 85 раз.**

Воспользуемся формулой Бернулли: $\displaystyle P_{n}(X=k)=C^k_{n}p^{k}q^{n-k}$, где:

$n=100\\k=85\\p=0.8\\q=0.2$ 

Тогда:  

$P_{100}(X=85)=C^{85}_{100}\cdot0.8^{85}\cdot0.2^{15}=\frac{100!}{85!(100-85)!}\cdot0.8^{85}\cdot0.2^{15}$   

Вычислим:

In [7]:
from math import factorial, exp

In [10]:
def bernulli(n, k, p):
    comb = factorial(n)/(factorial(k) * factorial(n - k))
    return comb * (p**k) * (1 - p)**(n - k)

In [14]:
print(f'P = {round(bernulli(100, 85, 0.8), 4)}')

P = 0.0481


---  

#### Ответ:  

$P(A)=0.0481=4.81\%$  

---

**2. Вероятность того, что лампочка перегорит в течение первого дня эксплуатации, равна 0.0004. В жилом комплексе после ремонта в один день включили 5000 новых лампочек.**  
   **a) Какова вероятность, что ни одна из них не перегорит в первый день?**  

Так как в задаче вероятность наступления события мала, а количество испытаний велико, для решения задачи воспользуемся формулой Пуассона:  

$\displaystyle P_m=\frac{\lambda^m}{m!}\cdot e^{-\lambda}\text{, где }\lambda \text{ среднее количество наступления события за определенную единицу измерений, и }\lambda=p\cdot n$

Напишем функцию:  

In [16]:
def poisson_distr(m,p,n):
    lamb = p * n
    return exp(-lamb) * (lamb**m) / factorial(m)

a)  По условию задания:

$\displaystyle p=0.0004\\n=5000\\m=0$  

Тогда:

In [17]:
print(f'P = {poisson_distr(2, 0.0004, 5000): .4f}')

P =  0.2707


---  
#### Ответ:  

$\displaystyle P(A)\approx0.2707\approx27.07\%$  

---

**3. Монету подбросили 144 раза. Какова вероятность, что орел выпадет ровно 70 раз?**

Применим формулу Бернулли:

$n = 144\\k = 70\\p = 0.5$

Тогда:

In [20]:
print(f'P = {bernulli(144,70,0.5): .4f}')

P =  0.0628


---  
#### Ответ:  

$\displaystyle P(A)\approx0.0628\approx6.28\%$  

---